n_t_with_s/Snakefile

```diff
@@ -0,0 +1,386 @@
+
+"""
+Snakefile for running N_t analysis on stdpopsim.
+
+Simply running `snakemake` will run all analysis 
+defined by the arguments above.
+
+Currently, for each rep,
+This will run stairway plot, smc++, and msmc
+on the data resulting from simulations
+on all chromosomes included in chrm_list
+for estimates of N_t (Ne through time).
+"""
+
+import pathlib
+import sys
+import os
+import numpy as np
+import stdpopsim
+import stairway
+import smc
+import msmc
+import plots
+
+
+# ###############################################################################
+# KNOBS - 
+# ###############################################################################
+
+
+# A seed to replicate results
+# TODO mutation rates
+
+configfile: config["config"]+"/config.json"
+
+np.random.seed(config["seed"])
+
+# This is the number of samples to simulate for within each population
+# for each replicate
+# TODO double check this is up to date with stdpopsim backend 
+num_samples_per_population = config["num_samples_per_population"]
+population_id = config["population_id"]
+# Here is a list of sample sizes to run msmc on. 
+# Each element counts as its own analysis
+# so there will be "replicates" runs for each size
+num_sampled_genomes_msmc = [int(x) for x in config["num_sampled_genomes_msmc"].split(",")]
+
+# The number of msmc Baumwelch(?) iterations to run,
+# typically 20
+num_msmc_iterations = config["num_msmc_iterations"]
+
+num_sampled_genomes_per_replicate = config["num_sampled_genomes_per_replicate"]
+
+# The number of replicates of each analysis you would like to run
+# For now leaving it a 1 just to get results quickly
+replicates = config["replicates"]
+
+# Define where selection will be imposed (whole genome or exons)
+selection_list = [sel for sel in config["selection"].split(",")]
+selection = selection_list[0]
```

Comment:
you don't need to parse this as a list if it's just parsing `exons` or `all`. change this to
```suggestion
selection = config["selection"]
```

Reply:
added change